src/components/HomePage.vue

```diff
@@ -136,6 +142,22 @@
     toggleToDoEditState();
   }
 
+  function filterToDos(filterCategory: string, order: number) {
+    if (filterCategory === '') {
+      return;
+    }
+    const currentToDosOnDisplay = displayItems.value.slice();
+    if (filterCategory === 'Title') {
+      displayItems.value = sortByTitle(currentToDosOnDisplay, order);
+    } else if (filterCategory === 'Text') {
+      displayItems.value = sortByText(currentToDosOnDisplay, order);
+    } else if (filterCategory === 'Date') {
+      displayItems.value = sortByDate(currentToDosOnDisplay, order);
+    } else if (filterCategory === 'Priority') {
+      displayItems.value = sortByPriority(currentToDosOnDisplay, order);
+    }
```

Comment:
Can you please find a solution to improve this function too without using if, else if, else if....